roles/easy-build/tasks/main.yml

```diff
@@ -0,0 +1,145 @@
+#
+# an ansible role to install easy_build.
+#
+# - name: "checking tmp directories"
+#   stat:
+#     path: "{{ eb_tmp_dir }}"
+#   register: folder_stats
+    
+# - name: "creating tmp folders"    
+#   file: 
+#     path: "{{ item.item }}"
+#     state: directory
+#     force: yes
+#     recurse: yes
+#     mode: 'u=rwx,g=rwx,o=rx' 
+#   when: item.stat.exists == false
+#   with_items:
+#     - "{{ folder_stats.results }}"
+---
+- name: "install needed packages"
+  become: yes
+  action: apt name=python-setuptools state=latest
+  when: ansible_os_family == "Debian"
+
+- name: "install needed packages"
+  become: yes
+  action: yum name=python-setuptools state=latest
+  when: ansible_os_family == "RedHat"
+
+- name: "rm previous tmp folders"    
+  file: 
+    path: "{{ eb_tmp_dir }}"
+    state: absent
+    force: yes
+
+- name: "rm EasyBuild software"
+  file: path="{{ eb_software_path }}/EasyBuild" state=absent force=yes
+
+- name: "rm EasyBuild profile file"
+  become: yes
+  file: path="/etc/profile.d/z01_eb.sh" state=absent force=yes
```

Comment:
Should use the extended format to be consistent with our other roles and the other tasks in this role